src/controllers/dashboardController.ts

```diff
@@ -0,0 +1,66 @@
+import prisma from '../config/prismaClient';
+import { Request, Response } from 'express';
+
+//returns userInfo for dashboard
+export const userInfo = async (req: Request, res: Response): Promise<unknown> => {
+  const user: any = req.user
+  if (user && user.id) {
```

Comment:
if isAuthenticated middleware is present, I think user and user.id would be defined, we don't need to put a check again..?